legacy/src/api/invitations.md

```diff
@@ -21,18 +21,18 @@ An Invitation can be used to allow users to claim ownership of a resource on the
 
 {% h4 Mandatory Fields %}
 
-|    Field     |        Type        |                               Description                                |
-| :----------- | :----------------- | :----------------------------------------------------------------------- |
-| id           | String             | The Invitation's unique identifier.                                      |
-| code         | String             | The Invitation code.                                                     |
-| type         | String             | The type of the related resource. Supported values are `kete-enrolment`. |
-| resourceId   | String             | The id of the related resource.                                          |
-| resourceType | String             | The type of the related resource. Supported values are `integration`.    |
-| expiresAt    | {% dt Timestamp %} | When the Invitation expires.                                             |
-| createdAt    | {% dt Timestamp %} | When the Invitation was created.                                         |
-| createdBy    | {% dt CRN %}       | The User or API Key that created the Invitation.                         |
-| updatedAt    | {% dt Timestamp %} | When the Invitation was updated.                                         |
-| updatedBy    | {% dt CRN %}       | The User or API Key that updated the Invitation.                         |
+|    Field     |        Type        |                                    Description                                       |
+| :----------- | :----------------- | :----------------------------------------------------------------------------------- |
+| id           | String             | The Invitation's unique identifier.                                                  |
+| code         | String             | The Invitation code.                                                                 |
+| type         | String             | The type of invitation. Supported values are `kete-enrolment`, `account-membership`. |
```

Comment:
I wonder if we should move `kete-enrolment`, `account-membership` out of the table? We could create a seperate table with these values in it.